src/index.css

```diff
@@ -1,4 +1,206 @@
-body {
+.Bookstore-CMS {
```

Comment:
- [ ] Your app looks very cool. However, all elements in design should be added and the styling was supposed to be EXACTLY the same as provided on Zeplin as mentioned also in the project requirements. I have marked the differences with yellow color in the below screenshot. For example, there are the following differences:
1. navbar `icon` which is on the right side is not the same as provided on Zeplin;
2. there should be `Comments | Remove | Edit ` right below the book author name;
3. on the right side of the section which is for displaying the books, there should be 2 sections separated with a `bar`, the first one shows a progress bar and the second one shows `Current Chapter`.
- Your design 👇 
![Screenshot (206)_LI](https://user-images.githubusercontent.com/67087664/155701382-a435afbc-c93d-4601-8f3d-1868c481600c.jpg)
- Zeplin design 👇 
![Screenshot (202)](https://user-images.githubusercontent.com/67087664/155701432-24ac2215-ba0d-4283-907f-3d6f1cefc607.png)
Kindly implement the changes to match the given design 🙏